packages/wiggle/src/BigWigAdapter/BigWigAdapter.ts

```diff
@@ -5,7 +5,7 @@ import { openLocation } from '@gmod/jbrowse-core/util/io'
 import { ObservableCreate } from '@gmod/jbrowse-core/util/rxjs'
 import SimpleFeature, { Feature } from '@gmod/jbrowse-core/util/simpleFeature'
 import AbortablePromiseCache from 'abortable-promise-cache'
-import QuickLRU from 'quick-lru'
+import QuickLRU from '@gmod/jbrowse-core/util/QuickLRU'
```

Comment:
It doesn't compile when I use it:

![image](https://user-images.githubusercontent.com/25592344/67901253-fab76800-fb2b-11e9-9cf0-48ba36247d96.png)

There may be some workaround, since it did work before, but quick-lru does some weird mixing of ES6 and CommonJS, so any workaround would probably be a hack. It was easier to just do make my own LRU.

Reply:
Right...I actually asked about this https://github.com/sindresorhus/quick-lru/pull/12

I dunno why they decided to do this.. this import syntax is not common currently (e,g, `import quickLru = require('quick-lru');`) I guess this is ok for now. If it is a literal copy of quick-lru please add a header. Otherwise we could downgrade to quick-lru v3 for now and avoid this mess...

Reply:
By header I mean attribution

Reply:
I got the same problem with v3, so I added an attribution to ours.